app-admin/passook/passook-20121001.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit prefix
+
+DESCRIPTION="Password generator capable of generating pronounceable and/or secure passwords"
+HOMEPAGE="https://github.com/mackers/passook"
+# snapshot of https://github.com/mackers/passook.git
+SRC_URI="mirror://gentoo/${P}.tar.gz"
+
+S="${WORKDIR}/${PN}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 hppa ppc ppc64 x86 ~amd64-linux ~x86-linux ~ppc-macos"
+
+RDEPEND="dev-lang/perl
+	sys-apps/miscfiles"
+
+src_prepare() {
+	cd "${S}"
+	eapply "${FILESDIR}"/passook.patch
```

Comment:
Please use `PATCHES` as an array instead in global scope. No need for eapply or `cd`.